packages/coinstac-api-server/tests/resolvers.test.js

```diff
@@ -0,0 +1,1027 @@
+        },
+        computations: [COMPUTATION_IDS[0]],
+        inputMap: {
+          start: {
+            value: 1,
+          },
+          covariates: {
+            ownerMappings: [{
+              name: 'mapping-1',
+            }],
+          },
+        },
+      }],
+    },
+  };
+
+  let error = await Mutation.savePipeline({}, args2);
+  t.is(getMessageFromError(error), INVALID_PIPELINE_STEP);
+
+  /* deletePipeline */
+  const pipelines = await Query.fetchAllPipelines();
+  const pipelineId = pipelines[0].id;
+  const consortiumId = pipelines[0].owningConsortium;
+
+  const invalidPermission = {
+    consortia: {
+      [consortiumId]: null,
+    },
+  };
+
+  error = await Mutation.deletePipeline(
+    { auth: { credentials: { permissions: invalidPermission } } }, { pipelineId }
+  );
+  t.is(getMessageFromError(error), ACTION_PERMIT_ERROR);
+
+  const permissions = {
+    consortia: {
+      [consortiumId]: ['owner'],
+    },
+  };
+
+  error = await Mutation.deletePipeline(
+    { auth: { credentials: { permissions } } }, { pipelineId }
+  );
+  t.is(getMessageFromError(error), RUNS_ON_PIPELINE);
+
+  error = await Mutation.deletePipeline(
+    { auth: { credentials: { permissions } } }, { pipelineId: PIPELINE_IDS[1] }
+  );
+  // t.is(getMessageFromError(error), RUNS_ON_PIPELINE);
+});
+
+test('updateRunState', async (t) => {
+  const args = {
+    runId: RUN_IDS[0],
+    data: {
+      state: 'running',
+    },
+  };
+
+  const res = await Mutation.updateRunState({}, args);
+
+  t.deepEqual(res.remotePipelineState, args.data);
+});
+
+test('updateUserConsortiumStatus', async (t) => {
+  const auth = getAuth(USER_IDS[0], 'test1');
+  const consortiumId = CONSORTIA_IDS[1];
+  const status = 'completed';
+
+  const res = await Mutation.updateUserConsortiumStatus(auth, { consortiumId, status });
+
+  t.is(res.consortiaStatuses[consortiumId], status);
+});
+
+test('updateConsortiumMappedUsers and updateConsortiaMappedUsers', async (t) => {
+  /* updateConsortiumMappedUsers */
+  let args = {
+    consortiumId: CONSORTIA_IDS[0],
+    mappedForRun: [USER_IDS[0], USER_IDS[1]],
+  };
+
+  let res = await Mutation.updateConsortiumMappedUsers({}, args);
+
+  t.deepEqual(res.mappedForRun, args.mappedForRun);
+
+  /* updateConsortiaMappedUsers */
+  const auth = getAuth(USER_IDS[0], 'test1');
+
+  res = await Mutation.updateConsortiaMappedUsers(auth, { consortia: null });
+  t.is(res, undefined);
+
+  args = {
+    consortia: [CONSORTIA_IDS[0]],
+  };
+
+  await Mutation.updateConsortiaMappedUsers(auth, args);
+});
+
+test('updatePassword', async (t) => {
+  const auth = getAuth(USER_IDS[0], 'test1');
+
+  let args = {
+    currentPassword: 'admin',
+    newPassword: 'admin123',
+  };
+
+  const res = await Mutation.updatePassword(auth, args);
+  t.is(getMessageFromError(res), INVALID_PASSWORD);
+
+  args = {
+    currentPassword: 'password',
+    newPassword: 'admin',
+  };
+
+  await Mutation.updatePassword(auth, args);
```

Comment:
should there be a test after this? Why not check if the password was updated via validate?